addons/web/static/tests/views/kanban_tests.js

```diff
@@ -5862,6 +5862,59 @@ QUnit.module('Views', {
         kanban.destroy();
     });
 
+    QUnit.test('"false" bar is clickable after removing filter', async function (assert) {
```

Comment:
This test must get splitted into:
- "active progressbar filter is kept and no non-matching records are added when switching to another view then get back in breadcrumbs"
- "active progressbar filter is kept and no non-matching records are added when adding then removing a filter whose addition had no matching record to the progressbar filter"

Okay, these are really long specific names, maybe you could rename them, but they express the intent you should reproduce in the tests.

Reply:
A little precision: this is not only a "_ false" value issue IMO.